assets/js/app.js

```diff
@@ -461,9 +459,14 @@ function getCoordinates( google, map ) {
                 })(currentMarker, i));
             }
         }
-    );
+    )
+        // clusters
+        .done(function( data ) {
+            if (typeof markers !== 'undefined' && markers.length > 0) {
+                var markerCluster = new MarkerClusterer(map, markers, {imagePath: 'https://developers.google.com/maps/documentation/javascript/examples/markerclusterer/m', maxZoom: "15"});
```

Comment:
`Uncaught ReferenceError: MarkerClusterer is not defined`

Reply:
Dabar veikia, bet panasu, kad filtruojant pagal tipus nesikeicia clusteriai.
Paziurek kaip padaryta cia https://codepen.io/obojdi/pen/KdNjOj ir pamegink pritaikyti

Reply:
ok